src/Polly.Core/Simmy/Latency/LatencyStrategyOptions.cs

```diff
@@ -7,13 +7,18 @@
 /// </summary>
 public class LatencyStrategyOptions : MonkeyStrategyOptions
```

Comment:
Could/should we mark any of these as `sealed`?

Reply:
Let's keep them open to be aligned with other options. For example `Microsoft.Extensions.Http.Resilience` derives from our options and changes some defaults. We want the same for new chaos options.

Reply:
In case of Microsoft.Extensions.Http.Resilience:
https://github.com/dotnet/extensions/blob/62c78eee58287a6030c13ce3e4f21190506d7291/src/Libraries/Microsoft.Extensions.Http.Resilience/Polly/HttpTimeoutStrategyOptions.cs#L11

The `HttpTimeoutStrategyOptions` derives from the `TimeoutStrategyOptions`.

I think we should keep them "open" for the same purpose.